### PR TITLE
Add Public OSS Demo 2 summary

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Add Public OSS Demo 2 summary

### CODE REVIEW

**Review by Progi1984 — APPROVED** [review]

File: news/_posts/public-oss-demo/2021-02-25-public-oss-demo-2.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+This setup was a compromise between the capacities of the tools, the quality of the video and the flexibility for maintainers.
+
+Attendees were able to ask questions in the chat that could be read and answered live. The streaming event has been recorded by YouTube and can be watched by the people who could not attend to the live stream.
+
+## This session agenda
+
+The session agenda was as follows:
+
+- 00:00 Waiting for everybody to join
```
Comment (PierreRambaud) [inline news/_posts/public-oss-demo/2021-02-25-public-oss-demo-2.md]:
```suggestion
- [00:00](https://youtu.be/JBvJgoo-rLg) Waiting for everybody to join
```

File: news/_posts/public-oss-demo/2021-02-25-public-oss-demo-2.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+- 02:34 Beginning of the session, introduction
```
Comment (PierreRambaud) [inline news/_posts/public-oss-demo/2021-02-25-public-oss-demo-2.md]:
```suggestion
- [02:34](https://youtu.be/JBvJgoo-rLg?t=154) Beginning of the session, introduction
```

File: news/_posts/public-oss-demo/2021-02-25-public-oss-demo-2.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+Attendees were able to ask questions in the chat that could be read and answered live. The streaming event has been recorded by YouTube and can be watched by the people who could not attend to the live stream.
+
+## This session agenda
+
+The session agenda was as follows:
+
+- [00:00](https://youtu.be/JBvJgoo-rLg) Waiting for everybody to join
+- 02:34 Beginning of the session, introduction
+- 04:48 Presentation of the agenda
```
Comment (PierreRambaud) [inline news/_posts/public-oss-demo/2021-02-25-public-oss-demo-2.md]:
```suggestion
- [04:48](https://youtu.be/JBvJgoo-rLg?t=288) Presentation of the agenda
```

File: news/_posts/public-oss-demo/2021-02-25-public-oss-demo-2.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+Attendees were able to ask questions in the chat that could be read and answered live. The streaming event has been recorded by YouTube and can be watched by the people who could not attend to the live stream.
+
+## This session agenda
+
+The session agenda was as follows:
+
+- [00:00](https://youtu.be/JBvJgoo-rLg) Waiting for everybody to join
+- [02:34](https://youtu.be/JBvJgoo-rLg?t=154) Beginning of the session, introduction
+- [04:48](https://youtu.be/JBvJgoo-rLg?t=288) Presentation of the agenda
+- 05:23 Event rules
```
Comment (PierreRambaud) [inline news/_posts/public-oss-demo/2021-02-25-public-oss-demo-2.md]:
```suggestion
- [05:23](https://youtu.be/JBvJgoo-rLg?t=323) Event rules
```

File: news/_posts/public-oss-demo/2021-02-25-public-oss-demo-2.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+- 06:12 Project announcements
```
Comment (PierreRambaud) [inline news/_posts/public-oss-demo/2021-02-25-public-oss-demo-2.md]:
```suggestion
- [06:12](https://youtu.be/JBvJgoo-rLg?t=372) Project announcements
```

File: news/_posts/public-oss-demo/2021-02-25-public-oss-demo-2.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+- 07:19 First demonstration - Product Page BO architecture by [@jolelievre](http://github.com/jolelievre)
```
Comment (PierreRambaud) [inline news/_posts/public-oss-demo/2021-02-25-public-oss-demo-2.md]:
```suggestion
- [07:19](https://youtu.be/JBvJgoo-rLg?t=439) First demonstration - Product Page BO architecture by [@jolelievre](http://github.com/jolelievre)
```

File: news/_posts/public-oss-demo/2021-02-25-public-oss-demo-2.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+- 33:39 Second demonstration - Multiple fixes for upgrade from PS 1.6 to PS 1.7.7 by [@atomiix](http://github.com/atomiix)
```
Comment (PierreRambaud) [inline news/_posts/public-oss-demo/2021-02-25-public-oss-demo-2.md]:
```suggestion
- [33:39](https://youtu.be/JBvJgoo-rLg?t=2019) Second demonstration - Multiple fixes for upgrade from PS 1.6 to PS 1.7.7 by [@atomiix](http://github.com/atomiix)
```

File: news/_posts/public-oss-demo/2021-02-25-public-oss-demo-2.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+- 45:40 Questions & Answers
```
Comment (PierreRambaud) [inline news/_posts/public-oss-demo/2021-02-25-public-oss-demo-2.md]:
```suggestion
- [45:40](https://youtu.be/JBvJgoo-rLg?t=2740) Questions & Answers
```

File: news/_posts/public-oss-demo/2021-02-25-public-oss-demo-2.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+- 54:08 Closure
```
Comment (PierreRambaud) [inline news/_posts/public-oss-demo/2021-02-25-public-oss-demo-2.md]:
```suggestion
- [54:08](https://youtu.be/JBvJgoo-rLg?t=3248) Closure
```

**Review by PierreRambaud — APPROVED** [review]
Thank @matks :heart:

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @matks

**Pull request merged**